### ISSUE TITLE
Additional sort orders

### ISSUE THREAD

**Issue by escowles** [pulibrary/pomegranate#307]
Add ability to sort search results by 
- title
- date
- author
- place of publication

**Comment by hackmastera** [comment]
My memory of solr is that you can only sort on a single-valued field. Is this no longer the case? If it is the case we'll need single-value versions of some/all of these and a way to decide which value to use.

**Comment by tpendragon** [comment]
That's still the case.

**Comment by escowles** [comment]
Yes, I think we should make sure we're always using the Roman-script titles to make the sorting more consistent.

**Comment by hackmastera** [comment]
Is there a way to tell what language a title is in?

**Comment by tpendragon** [comment]
You can tell the direction it's in (`#dir` I think?), but I think that's it.

**Comment by tpendragon** [comment]
The source metadata has the language info, so we can just make sure the first or last thing is the romanized title when we index.

**Comment by hackmastera** [comment]
I can't find any example records with publication_place data. does anyone know of one?

**Comment by tampakis** [comment]
Maybe publication place doesn't make it into figgy? Here's a record in figgy that has a "publication_place_facet" value in Orangelight https://bibdata.princeton.edu/bibliographic/4606238/solr
https://figgy.princeton.edu/catalog/3addcc26-6800-4aa3-b713-35e70483bc94

**Comment by hackmastera** [comment]
Yeah it's available as a MARC relator field but I'm not sure anything is using it. I'm not sure where the mapping is defined; maybe as part of the imported metadata logic which I didn't dive deeply into.

**Comment by hackmastera** [comment]
We've agreed to move forward without place_of_publication. It seems like a use case well-served by searching, and the data doesn't really seem to be there to support it. If we find a good use case to support doing this, after all, we should open a new ticket for that work.

**Comment by hackmastera** [comment]
for deploy: each collection must be reindexed. Once reindex is complete, go to dashboard configuration > search > results, select the new sort options, and save. then they will appear.

@escowles this is currently deployed to pomegranate staging and on for the Cotsen collection.

**Comment by hackmastera** [comment]
except that it's not working. will look into it.

**Comment by hackmastera** [comment]
reverted the commit because things weren't working on staging. will re-test pending new staging box, which is block by ruby version work in the ansible repo.
